fixtures/attribute-behavior/src/attributes.js

```diff
@@ -573,6 +573,24 @@ const attributes = [
     tagName: 'path',
     read: getSVGAttribute('externalResourcesRequired'),
   },
+  {
+    name: 'fetchPriority',
+    overrideStringValue: 'high',
```

Comment:
@eps1lon Why would this override?

Reply:
Just so that we test with a valid value instead of an arbitrary one that you'd never use in a real application